shelley/chain-and-ledger/executable-spec/test/Test/Shelley/Spec/Ledger/Serialization.hs

```diff
@@ -246,12 +248,12 @@ checkEncoding ::
   TestTree
 checkEncoding encode decode name x t =
   testCase testName $
-    assertEqual testName (fromEncoding $ toCBOR t) (fromEncoding $ encode x)
+    assertEqual testName
+      (Base16.encode $ serialize t)
+      (Base16.encode . serializeEncoding . encode $ x)
```

Comment:
this makes it _a lot_ easier to debug failures. you can now just dump the failure into cbor.me.